schema/site_metadata.json

```diff
@@ -50,6 +58,59 @@
             "type": "number"
         }
       }
+    },
+    "area": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "m2": {
+            "description": "Area of the site in square meters",
+            "type": "number",
+            "examples": [1234, 256, 998765]
+        }
+      }
+    },
+    "source": {
+      "type": "object",
+      "additionalProperties": false,
+      "properties": {
+        "folder": {
+          "description": "Folder where the UDMI site model is stored",
+          "type": "string",
+          "examples": ["/home/user/udmi_site_model", "https://drive.google.com/corp/drive/folders/iamadrivelocation"]
```

Comment:
NB: We should figure out the right canonical syntax to this... e.g. should it be file://home/user/udmi_site_model for a file?